core/src/main/java/me/mykindos/betterpvp/core/combat/combatlog/CombatLog.java

```diff
@@ -72,7 +72,7 @@ public void onClicked(Player player) {
 
         UtilServer.callEvent(new PlayerClickCombatLogEvent(player, this));
 
-        UtilMessage.broadcast("Log", "<yellow>%s</yellow> dropped their inventory for combat logging.", playerName);
+        UtilMessage.broadcast("Log", "<yellow>%s</yellow> caused <yellow>%s</yellow> dropped their inventory for combat logging.", player.getName(), playerName);
```

Comment:
missing a word here

Reply:
actually probably just need to re-word the last part of the sentence

Reply:
yeah